backend/src/main/java/ca/bc/gov/hlth/hnweb/config/WebClientConfig.java

```diff
@@ -113,6 +128,24 @@ public WebClient hcimWebClient() throws HNWebException {
                 .build();
     }
 
+	@Bean("hibcWebClient")
+    public WebClient hibcWebClient() throws HNWebException {
+
+		SslContext sslContext = getSSLContext(hibcUrl, hibcCertFile, hibcCertPassword);
+		
+	    HttpClient httpClient= HttpClient.create().secure(t -> t.sslContext(sslContext));
+		ClientHttpConnector connector= new ReactorClientHttpConnector(httpClient);
+	    
+		return WebClient.builder()
+	    		.clientConnector(connector)
+                .baseUrl(hibcUrl)
+                .filter(logRequest())
+                .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN_VALUE) 
+                .defaultHeader(HttpHeaders.ACCEPT, MediaType.ALL_VALUE) 
+                .defaultHeaders(header -> header.setBasicAuth(hibcUserName, hibcUserPassword))
```

Comment:
Passwords are different for relative url for different HIBC messages, need to move auth header to post call.

Reply:
Fixed. We can roll out this pattern to R15 and R50 as we implement them.